features/org.eclipse.e4.rcp/feature.xml

```diff
@@ -294,6 +294,10 @@
          arch="aarch64"
          version="0.0.0"/>
 
+   <plugin
+         id="org.eclipse.swt.svg"
+         version="0.0.0"/>
+
```

Comment:
In order to include the org.eclipse.swt.svg fragment in the provided Eclipse products this part is important.

As far as I can tell no code changes are necessary in Jface to make SVGs usable in Eclipse, aren't they?
Or to ask the other way, round: Would it make sense to just submit this part now?

Reply:
The only change required is the adaptation of the `URLImageDataProvider`, which directly instantiates `ImageData` in a non-zoom-aware way. This needs to be enhanced in order to avoid that SVGs are rasterized at 100% and then auto-scaled.

Reply:
As mentioned [here](https://github.com/eclipse-platform/eclipse.platform.swt/issues/1944#issuecomment-2753513977) I don't think dedicated inclusion in a feature is the right approach and instead a requirement should be added to bundles containing SVGs.

Reply:
Lets not have that discussion at multiple places. :) TLDR; I think this is fine:
- https://github.com/eclipse-platform/eclipse.platform.swt/issues/1944#issuecomment-2753574505

Reply:
I don't think its fine to introduce yet another implicit dependency. Just make it explicit and it will work everywhere.

Reply:
@HannesWell can this be deleted now?

Reply:
yes, please remove this for now

Reply:
done.